scipy/signal/_peak_finding.py

```diff
@@ -836,10 +846,10 @@ def find_peaks(x, height=None, threshold=None, distance=None,
     * For several conditions the interval borders can be specified with
       arrays matching `x` in shape which enables dynamic constrains based on
       the sample position.
-    * The order of arguments given in the function definition above mirrors the
-      actual order in which conditions are evaluated. In most cases this order
-      is the fastest one because faster operations are applied first to reduce
-      the number of peaks that need to be evaluated later.
+    * The conditions are evalutated in the following order: `plateau_size`,
+      `height`, `threshold`, `distance`, `prominence`, `width`. In most cases
+      this order is the fastest one because faster operations are applied
+      first to reduce the number of peaks that need to be evaluated later.
```

Comment:
Would this be more readable as a list?
```
* The conditions are ...:
  1. `plateau_size`
  2. `height`
  ...
```
Might be worth building doc locally to check and push if so.

Reply:
It's more readable. But considering that this is a trivia that won't matter for all but the most performance intensive use cases I would have preferred the more compact version that takes up less vertical space. But that's not a strong opinion. Your decision. ;)

Fulltext:
![image](https://user-images.githubusercontent.com/20140352/45308555-014ae080-b522-11e8-9abb-08f061e45ff3.png)

As a list:
![image](https://user-images.githubusercontent.com/20140352/45308290-6d791480-b521-11e8-9f9e-7222332ff762.png)

Reply:
Ahh right it's just a performance consideration not something that actually changes the output. Either way is fine with me, then, up to you.